jujugui/static/gui/src/app/components/deployment-bar/test-deployment-bar.js

```diff
@@ -62,6 +62,16 @@ describe('DeploymentBar', function() {
           tabIndex="0">
           Export
         </span>
+        <span className="deployment-bar__import link"
```

Comment:
You'll need to add a test which checks to see that your new functionality works.

- Stub the props
- Set the value of the input field
- Simulate a click on the span
- Check that the props were called with the proper data.